main/src/io/github/iltotore/iron/constraint/string.scala

```diff
@@ -49,6 +49,11 @@ object string:
   type UUIDLike =
     Match["^([0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12})"] DescribedAs "Should be an UUID"
 
+  /**
+   * Tests if the input contains at least 1 non-whitespace character
+   */
+  final class NonEmpty
```

Comment:
Rename to `NotOnlyWhitespaces` or similar.

Reply:
Upon some more thought, may I suggest `NotBlank` instead? This is the term used in Java (see [here](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#isBlank())). I'm really not sure why I didn't use this term first since my initial implementation was with this method, but I ended up not using it since it's only available in Java 11+.

Reply:
`NotBlank` is perfect. In fact, I wasnt satisfied either of `NotOnlyWhitespace`